common.blocks/dropdown/dropdown.js

```diff
@@ -86,7 +86,10 @@ provide(bemDom.declBlock(this.name, /** @lends dropdown.prototype */{
         this.setMod('opened', data.modVal);
     }
 }, /** @lends dropdown */{
-    lazyInit : true
+    lazyInit : true,
+    onInit : function() {
+        this._events(Switcher).on('click', this.prototype.onSwitcherClick);
```

Comment:
actually there is: https://github.com/bem/bem-components/blob/96736547f914964c1652db26560f987b241b6611/common.blocks/dropdown/dropdown.js#L80